lib/bap_powerpc/bap_powerpc_dsl.mli

```diff
@@ -0,0 +1,91 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_powerpc_rtl
+
+type 'a ec
+
+type bitwidth
+
+val bit  : bitwidth
+val byte : bitwidth
+val halfword : bitwidth
+val word : bitwidth
+val doubleword : bitwidth
+val quadword : bitwidth
+val bitwidth : int -> bitwidth
+val int_of_bitwidth : bitwidth -> int
+
+val imm : (op -> exp) ec
+val var : (bitwidth -> exp) ec
+val reg : (reg -> exp) -> (op -> exp) ec
+val const : (bitwidth -> int -> exp) ec
+val of_string : (string -> exp) ec
+
+val signed : 'a ec -> 'a
+val unsigned : 'a ec -> 'a
+
+val zero : exp
+val one  : exp
+
+(** [extract e lx rx] extracts portion of [e] starting
+    at bit [lx] and ending at bit [rx], all bounds
+    are inclusive. Bits indexes start from the most
+    significant bit. *)
```

Comment:
do not duplicate documentation, there should be only one single point of truth. Please erase all duplicating comments (unless they are internal for the package)

Reply:
yes, they are for internal usage in the case of `Bap_powerpc_dsl` module